packages/ui/core-components/src/lib/unsorted/ui/Embed.svelte

```diff
@@ -0,0 +1,46 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	import { cn } from '$lib/utils.js';
+	import { toBoolean } from '../../utils.js';
+
+	let className = undefined;
+	export { className as class };
+
+	export let url = ''; // The URL to embed
+	export let title = ''; // A description or title for the embed
+	export let width = '100%'; // Width of the embed, defaults to full width
+	export let height = '400'; // Height of the embed, defaults to 400px
+	export let align = 'left'; // center, left, right
+	export let border = true; // Toggle border visibility
+	border = toBoolean(border);
+
+	// Process dimensions: Add `px` if a number, pass through if a valid unit
+	function processDimension(dimension) {
```

Comment:
If this was extracted out (which it appears to have been from other components), the definition shouldn't live here